server/routes/api/v1/adoptablePetsRouter.js

```diff
@@ -1,29 +1,46 @@
-import express from "express"
-
-import AdoptablePet from "../../../models/AdoptablePet.js"
-
-const adoptablePetsRouter = new express.Router()
-
-adoptablePetsRouter.get("/:type", async (req, res) => {
-  try {
-    const adoptablePets = await AdoptablePet.findByType(req.params.type)
-    res.status(200).json({ adoptablePets })
-  } catch (error) {
-    console.log("Router Error")
-    console.error(error)
-    res.status(500).json({ errors: error })
-  }
-})
-
-adoptablePetsRouter.get("/:type/:id", async (req, res) => {
-  try {
-    const adoptablePet = await AdoptablePet.findById(req.params.id)
-    res.json({ adoptablePet })
-  } catch (error) {
-    console.log("Router Error")
-    console.error(error)
-    res.status(500).json({ errors: error })
-  }
-})
-
-export default adoptablePetsRouter
+import express from "express"
+
+import AdoptablePet from "../../../models/AdoptablePet.js"
+import AdoptionApplication from "../../../models/AdoptionApplication.js"

```

Comment:
This router has the same name as a different router and has duplicate routes. A router should be specific for the resource it's dealing with to create a separation of concerns. When we are going to be interacting with an `adoptable pet` resource, all requests go through the `adoptable pets router`. If we are dealing with types of pets, that goes through a `pet types router`